lib/wallet/http.js

```diff
@@ -466,6 +466,10 @@ class HTTP extends Server {
       const outputs = valid.array('outputs', []);
       const sign = valid.bool('sign', true);
 
+      let changeAddress = valid.str(['changeAddress', 'change']);
```

Comment:
If I understand correctly, both `changeAddress` and `change` properties in the JSON body can be used. What happens if they are both included?

Reply:
Looks like it will use the first option, i.e. `changeAddress`.  Would you prefer if i just use one option?

Reply:
Would be good to not add ambiguity. I think `changeAddress` is intuitive and explicit, I'd go with just that.

Reply:
Ok, done.